sqlserver/datadog_checks/sqlserver/activity.py

```diff
@@ -61,6 +61,7 @@
     c.client_tcp_port as client_port,
     c.client_net_address as client_address,
     sess.host_name as host_name,
+    sess.program_name as program_name,
```

Comment:
Are there any existing tests we can piggy back on to assert we're collecting program_name and/or sending it in our collected activity?

Reply:
Please confirm this is available in all supported versions as well 🙏

Reply:
> Please confirm this is available in all supported versions as well 🙏
Yep! It's available on supported version.

> Are there any existing tests we can piggy back on to assert 
Doesn't look like we have a test case that asserts all the columns we collect besides some of the primary ones we care about. I can probably add onto it, but yeah, that would be a nice test to have.

Reply:
the column is there since at least 2016 https://learn.microsoft.com/en-us/sql/relational-databases/system-dynamic-management-views/sys-dm-exec-sessions-transact-sql?view=sql-server-2016